style.css

```diff
@@ -0,0 +1,97 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:ital@0;1&display=swap');
+
+
+body {
+  line-height: 1.4;
+  width: 100%;
+  min-width: 290px;
+  color: #4d4e53;
+  background-color: #ffffff;
+  font-family: 'Open Sans', Arial, sans-serif;
+  line-height: 1.5;
+}
+
+header {
+  text-align: left;
+  margin: 0px;
+  color: rgb(8, 8, 8);  
+  display: block;
+  font-weight: 600;
+  font-size: 1.8em;
+  line-height: 1.5;
+}
+.coder{
+  text-align: left;
+  display: block;  
+  white-space: pre-line;
+  position: relative;
+  line-height: 1;
+  background-color: #f7f7f7;
+  padding: 12px;
+  margin: 10px;
+  border-radius: 5px;
+}
+
+#navbar {
+  position: fixed;
+  height: 100%;
+  min-width: 300px;
+  width: 300px;
+  
+  top: 0px;
+  left: 0px;
+
+  color: black;
+  background-color: white;
+  padding: 75px 0px;
+  z-index: 3;
+}
+
+
+#navbar header {
```

Comment:
I think you leave much spaces in your line of codes

Reply:
> I think you leave much spaces in your line of codes


- [ ] You did a great job at using the inline comments, but I see that you didn't explain to the student why should work on the spaces, so please always tell the student the reason for the change
> For example: 😊⤵️
- You can here tell him/her that it will be better 😃 if you work on the spaces because maybe it damages the linter and makes the appearance of writing the code not good, so let's enhance the code by fixing the spaces of the statements and with that the code will be better and you will be a friend for the linter and passing it 😊💯